com.microsoft.mrtk.graphicstools.unity/Samples/Magnifier/Magnifier.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
All of these assets are in the "Samples" folder and not the "Samples~" folder. Can you fix that?

More details [here](https://docs.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/mrtk3-graphicstools/contributing#updating-the-package-samples).

Reply:
Also, can you move these assets into their respective Materials/Shaders/Scripts/etc. folders like the other samples?